scripts/validate_json.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+for FILE in `ls ./tests/books`
+do
+  jq . "tests/books/$FILE"
+  exit_code=$?
+    if [ $exit_code -ne 0 ]; then
+      echo "$FILE could not be parsed."
```

Comment:
You will probably then want to exit with the exit code that jq returned using `exit $exit_code`

Reply:
Done.